app/controllers/admin/applications_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Admin::ApplicationsController < ApplicationController
+
+  def show 
+    @application = Application.find(params[:id])
+   
+  end
+
+ 
+end
```

Comment:
A lot of extra spacing in the file that should probably be removed. You want the spacing in your code to be consistent across all files. I like to tell people that no matter how many people work on a project it should always "look" like one person wrote the whole thing because the styling is consistent throughout.